wee_slack.py

```diff
@@ -2266,32 +2267,37 @@ def refresh(self):
         self.rename()
 
     def mark_read(self, ts=None, update_remote=True, force=False):
-        if self.channel_buffer:
-            w.buffer_set(self.channel_buffer, "unread", "")
-            w.buffer_set(self.channel_buffer, "hotlist", "-1")
+        super().mark_read(ts=ts, update_remote=update_remote, force=force, post_data={"thread_ts": self.parent_message.ts})
```

Comment:
We still support Python 2, so you have to use `super(SlackThreadChannel, self)`.